main.go

```diff
@@ -165,12 +201,24 @@ func main() {
 		if _, ok := skipDependencies[d]; ok {
 			continue
 		}
-		c := []Code{
-			List(Id(toPrivateVar(d)), Err()).Op(":=").Qual(fmt.Sprintf("github.com/Clever/%s/gen-go/client", d), "NewFromDiscovery").Call(),
-			If(Err().Op("!=").Nil()).Block(
-				Qual("log", "Fatalf").Call(List(Lit("discovery error: %s"), Err())),
-			),
+		c := []Code{}
+
+		if replacementString, ok := overrideDependenciesMap[d]; ok {
+			c = []Code{
+				List(Id(toPrivateVar(d)), Err()).Op(":=").Qual(fmt.Sprintf("github.com/Clever/%s/gen-go/client", replacementString), "NewFromDiscovery").Call(),
+				If(Err().Op("!=").Nil()).Block(
+					Qual("log", "Fatalf").Call(List(Lit("discovery error: %s"), Err())),
+				),
+			}
+		} else {
+			c = []Code{
+				List(Id(toPrivateVar(d)), Err()).Op(":=").Qual(fmt.Sprintf("github.com/Clever/%s/gen-go/client", d), "NewFromDiscovery").Call(),
+				If(Err().Op("!=").Nil()).Block(
+					Qual("log", "Fatalf").Call(List(Lit("discovery error: %s"), Err())),
+				),
+			}
```

Comment:
It would be less code and equally clean if you have `importPackage, ok := overrideDependenciesMap[d]` and `if !ok {importPackage = d}` and keep the rest same just replace `d` in Sprintf with `importPackage`